### PR TITLE
Added Aggregate-backup guide

### LINKED ISSUE #101: Describe backup strategies for Aggregate

### PR DESCRIPTION
**Pull request by ankita240796**
closes #101 

## What is included in this PR?
- Added backup information from Briefcase.
- Added backup information from SQL dump.
- Added backup information from GAE.

### CODE REVIEW

**Comment by ankita240796** [comment]
@adammichaelwood build is failing because [Sphinx generates warning](https://github.com/sphinx-doc/sphinx/issues/2229) on using any unknown option with the `:option:` directive. We can [suppress these warnings](https://github.com/sphinx-doc/sphinx/pull/2346) but I am not sure what to do here. Can you please guide me on this.

**Comment by ankita240796** [comment]
@adammichaelwood build is failing because [Sphinx generates warning](https://github.com/sphinx-doc/sphinx/issues/2229) on using any unknown option with the `:option:` directive. We can [suppress these warnings](http://www.sphinx-doc.org/en/stable/config.html#confval-suppress_warnings) but I am not sure what to do here. Can you please guide me on this.

**Comment by adammichaelwood** [comment]
>  because Sphinx generates warning on using any unknown option 

What a terrible feature. 

We definitely want to suppress this. 
If there is a way to suppress in `conf.py`, please add that to this PR. -- That's the best way.

Otherwise, if it can only be suppressed on the command line, please submit a separate PR that adds the option to the circle CI file and `b.sh`, and flag @yanokwa to review it

**Comment by ankita240796** [comment]
@adammichaelwood I added warning suppression for unknown options in `conf.py`.

File: aggregate-backup.rst [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+---------------------------
+
+Use :doc:`ODK Briefcase <briefcase-guide>` to back up all forms and submissions on Aggregate. You can :ref:`pull forms from your Aggregate server <pull-from-aggregate>` into your local machine using Briefcase. In particular, Briefcase's :ref:`command line interface <cli-use>` makes this easier.
+
+.. _mysql-backup:
+
+Recovering database from MySQL dump 
+--------------------------------------
+
+1. Stop running the Tomcat.
```
Comment (adammichaelwood) [inline aggregate-backup.rst]: "the Tomcat" should be either "Tomcat" or "the Tomcat server"

File: aggregate-backup.rst [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+2. Upgrade to the latest version og Aggregate.
```
Comment (adammichaelwood) [inline aggregate-backup.rst]: typo

File: aggregate-backup.rst [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+3. Finally restore it from MySQL dump. An SQL dump of a database is a common method to safely store away a snapshot of the database for archival purposes or to migrate data between database instances, e.g. between two major system releases. The content of a SQL dump is a large collection of SQL commands in ASCII. Running the script will recreate the database in the same state as it was when the dump was created. The primary tool to consider for making an ASCII dump is msqldump, which includes a wide variety of options:
```
Comment (adammichaelwood) [inline aggregate-backup.rst]: link to msqldump

File: aggregate-backup.rst [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+
+.. code-block:: console
+
+	$ msqldump [ options ] [ dbname ]
+
+Options are:
```
Comment (adammichaelwood) [inline aggregate-backup.rst]: i don't think we need msqldump docs recreated here. Mention the commands/options our folks are going to use and then link to the msqldump documentation

**Comment by ankita240796** [comment]
@adammichaelwood I made the requested changes.

**Comment by adammichaelwood** [comment]
@ankita240796 i took care of the marge conflict

**Comment by ankita240796** [comment]
Thank you @adammichaelwood!

**Pull request merged**